workflow-templates/rebase.yml

```
@@ -0,0 +1,18 @@
+on:
+    issue_comment:
+        types: [ created ]
+name: Automatic Rebase
+jobs:
+    rebase:
+        name: Rebase
+        if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
```

There was a few safety checks I added.
```suggestion
    # On pull requests and if the comment starts with `/compile`
    if: github.event.issue.pull_request != '' && startsWith(github.event.comment.body, '/rebase')
```

---

done